pyicloud/base.py

```diff
@@ -72,11 +71,11 @@ def request(self, method, url, **kwargs):  # pylint: disable=arguments-differ
         if self.service.password_filter not in request_logger.filters:
             request_logger.addFilter(self.service.password_filter)
 
-        request_logger.debug("%s %s %s" % (method, url, kwargs.get("data", "")))
+        request_logger.debug("{} {} {}".format(method, url, kwargs.get("data", "")))
```

Comment:
Please use logging formatting such that the parameters are not stringified unless the log level is low enough to print it.

```suggestion
        request_logger.debug("%s %s %s", method, url, kwargs.get("data", ""))
```